plugin_ide.ui/src/com/github/rustdt/ide/ui/editor/RustEditorActionContributor.java

```diff
@@ -28,6 +47,88 @@ protected RustOpenDefinitionOperation createOpenDefinitionOperation(ITextEditor
 	
 	@Override
 	protected void registerOtherEditorHandlers() {
+		setupOpenType();

```

Comment:
Not sure where to put that code. Here or in the `LangEditorActionContributor`? Or something else? The global type structure is not directly related to an editor, is it?